lib/ic/power/tps62132.yaml

```diff
@@ -0,0 +1,24 @@
+datasheet: http://www.ti.com/lit/ds/symlink/tps62132.pdf
+description: 3.3Vout 3A, fixed, Buck Converter
+footprint: agg:QFN-16-EP-TI
+name: TPS62132
+ordercodes:
+- [Farnell, '2492468']
+pins:
+- - - [PVIN, 11, pwrin]
+    - [PVIN, 12, pwrin]
+    - [AVIN, 10, pwrin]
+    - [EN, 13, pwrin]
+  - - [SS/TR, 9, passive]
+  - - [DEF, 8, in]
+    - [FB, 5, passive]
+    - [AGND, 6, pwrin]
+    - [PGND, 15, pwrin]
+    - [PGND, 16, pwrin]
+    - [EP, EP, pwrin]
+- - - [SW, 1, pwrout]
```

Comment:
given as SW always connects to an inductor it'l never get seen as a pwrout - probably best to have them all passive?